internal/pkg/utils/slice.go

```diff
@@ -6,6 +6,11 @@ import (
 	"sort"
 )
 
+type objectWithName interface {
+	ObjectName() string
+	String() string
+}
+
 // SortByName - in tests are IDs and sort random -> so we must sort by name.
 func SortByName(slice interface{}) interface{} {
```

Comment:
Tato metoda zoradzuje objekty v testoch, ... zistil som, ze s pouzitim interface sa to da napisat podstatne jednoduchsie.